eduNEXT/eox-nelp
# Jlc/add save succesful phone validated

## Description

feat: save_successfull_phone_validation in otp
## Testing instructions


Please configure you environment with the nelp_custom_reg form with this change.
https://github.com/nelc/nelp-custom-registration-fields/pull/4

- Test generate and validate endpoint.
`/api/one-time-password/v1/generate/`
`/api/one-time-password/v1/validate/`

- check the field of phone_validated  in extra_info is also updated.
[Screencast from 13-06-24 15:13:35.webm](https://github.com/eduNEXT/eox-nelp/assets/51926076/496c8ba7-03b2-46d4-b24a-78b851e8834b)

## Additional information

#### Jira story
https://edunext.atlassian.net/browse/FUTUREX-776

## Checklist for Merge
